gitlab/v4/objects/merge_request_approvals.py

```diff
@@ -111,6 +113,7 @@ def set_approvers(
             "rule_type": "regular",
             "user_ids": approver_ids,
             "group_ids": approver_group_ids,
+            "usernames": approver_usernames,
```

Comment:
Before around line 108 had:

```
approver_usernames = approver_usernames or []
```

We still need that. Otherwise `None` will be sent in the request.

Reply:
Bah, right, adding that back -- had took it out thinking a default  of [] would cover it, forgot to add it back.